pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTAnnotationTypeDeclaration.java

```diff
@@ -7,9 +7,8 @@
 
 import java.util.List;
 
-public class ASTAnnotationTypeDeclaration extends AbstractJavaAccessTypeNode implements ASTAnyTypeDeclaration {
+public class ASTAnnotationTypeDeclaration extends ASTAnyTypeDeclaration {
```

Comment:
this is a breaking API change. Do we really need to do it in 6.2.0? Can't we just deprecate the methods?

Reply:
Which methods?